enterprise/kerberos-login-module/pom.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>kerberos-login-module</groupId>
+    <artifactId>kerberos-login-module</artifactId>
+    <version>0.1-SNAPSHOT</version>
+    <name>Kerberos Login Module</name>
+    <description>A Kerberos Login Module for Hazelcast.</description>
+    
+    <!--
+    <packaging>jar</packaging>
+    -->
+
+    <parent>
+        <artifactId>enterprise</artifactId>
+        <groupId>com.hazelcast.samples.enterprise</groupId>
+        <version>0.1-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <properties>
+        <!-- needed for checkstyle/findbugs -->
+        <main.basedir>${project.parent.parent.basedir}</main.basedir>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+    
+    <build>
+      <plugins>
+        <plugin>
+          <!--
+          <groupId></groupId>
+          -->
+          <artifactId>kerberos-login-module</artifactId>
+          <version>${hazelcast.version}</version>
+          <configuration>
+            <archive>
+              <manifestFile>src/main/resources/META-INF/MANIFEST.MF</manifestFile>
+            </archive>
+          </configuration>
+        </plugin>
+      </plugins>
+    </build>
```

Comment:
Probably we don't need this part. Od do you implement some Maven plugin within the code?

Reply:
Realized we don't need it indeed, removing.